src/components/TodoList/TodoList.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { User } from '../User';
+import './TodoList.css';
+
+export const TodoList = ({ todos }) => (
+  <div className="list">
+    <ul>
+      {todos.map(todo => (
+        <li key={todo.id}>
+          <p>
+            <User user={todo.user} />
+          </p>
+          <p>
+            {`${todo.title} - `}
+            <span>
+              {todo.completed ? (
+                <span className="todo__status--done">Done</span>
```

Comment:
You could create only one `span` instead of two with text {todo.completed ? 'Done' : 'Doing'} and use `classnames` library for switching classes